enclave-tls/src/enclave_quotes/sgx-ecdsa/verify_evidence.c

```diff
@@ -44,23 +50,110 @@ enclave_quote_err_t sgx_ecdsa_verify_evidence(enclave_quote_ctx_t *ctx,
 		   quote_size, pquote->signature_data_len);
 
 	/* 1 means trusted verify methond by QvE, 0 means verify by untructed QPL */
-	bool verify_by_qve = 0;
+	bool verify_by_qve = 1;
```

Comment:
This parameter should be controlled by the configuration provided by enclave_tls_init()